FreeRTOS/FreeRTOS-Plus-TCP
# A new algorithm to select the endpoint for received packets[ PR #713 ]

A new algorithm to select the endpoint for received packets[ PR #713 ]

Description
-----------
1. Taking changes as added in PR#713 , removing files , which had only indentation changes
2. Adding similar change for the Zynq Network interface (removing hard coded value for single end point)

Test Steps
-----------
WinSim Echo Server and client Demo run steps

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
